cmd/commands/cli/command.go

```diff
@@ -58,7 +59,7 @@ const serviceHelp = `service <action> [args]
 // NewCommand constructs CLI based Mysterium UI with possibility to control quiting
 func NewCommand() *cli.Command {
 	return &cli.Command{
-		Name:   cliCommandName,
+		Name:   CliCommandName,
```

Comment:
Actually according to go bible this should be `CLICommandName` or `cliCommandName` as CLI is an acronym